include/swift/Option/SanitizerOptions.h

```diff
@@ -15,6 +15,7 @@
 
 #include "swift/Basic/Sanitizers.h"
 #include "llvm/ADT/Triple.h"
+#include "llvm/ADT/STLExtras.h"
```

Comment:
Nitpick: headers are out of order. (They should be alphabetical within the project.)